scripts/auto-update-and-commit.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -ex
+./src/update_versions_to_latest_master.py
+# https://stackoverflow.com/a/3879077/777939
+git diff-index --quiet HEAD -- && echo "No changes detected" && exit 0;
+echo "Changes detected"
+
+git remote set-url origin git@github.com:vimc/montagu
+git push --delete origin latest || true
+git branch -d latest || true
+git checkout -b latest
+
+git add submodules
+git commit -m "Auto: Update versions to latest"
+git push -u origin latest
```

Comment:
So as I understand it we have the ordinary deploy task running on all branches, but this one just running `master` right? So on the main deploy task we should trigger a staging machine after successful deploy iff the branch is `latest`. Do we already have a separate ticket for that?

Reply:
I've raised https://vimc.myjetbrains.com/youtrack/issue/VIMC-2123 for this.